Package.swift

```diff
@@ -2,23 +2,64 @@
 
 import PackageDescription
 
+let MeiliSearch = "MeiliSearch"
+let MeiliSearchCore = "MeiliSearchCore"
+let MeiliSearchNIO = "MeiliSearchNIO"
+
+var products: [PackageDescription.Product] = [
+  .library(name: MeiliSearch, targets: [MeiliSearch]),
+  .library(name: MeiliSearchCore, targets: [MeiliSearchCore])
+]
+
+var dependencies = [PackageDescription.Package.Dependency]()
+
+var targets: [PackageDescription.Target] = [
+  .target(
+    name: MeiliSearch,
+    dependencies: [
+      .target(name: MeiliSearchCore)
+    ]
+  ),
+  .target(
+    name: MeiliSearchCore, 
+    dependencies: []
+  ),
+  .testTarget(
+    name: "MeiliSearchUnitTests",
+    dependencies: ["MeiliSearch"]
```

Comment:
nit: where we reference target names, let's be consistent and use the static variable

```suggestion
    dependencies: [.target(MeiliSearch)]
```